src/langcheck/plot/_utils.py

```diff
@@ -0,0 +1,33 @@
+import re
+from enum import Enum
+
+from plotly.graph_objects import Figure
+
+
+class Axis(Enum):
+    vertical = 0
+    horizontal = 1
+
+
+def _plot_threshold(fig: Figure, threshold_text: str, direction: Axis):
+    # Analyize threshold condition
+    pattern = r'(<|<=|>|>=|==)\s([0-9]\.[0-9]+)'
+    match = re.search(pattern, threshold_text)
+    if match:
+        operator, threshold = match.groups()
+    else:
+        raise ValueError('Threshold not Found!')
+    operator, threshold = operator[0], float(threshold)
+    if direction == Axis.horizontal:  # Paint in row
```

Comment:
```suggestion
    if direction == Axis.horizontal:  # Draw a horizontal line
```
Is this what you meant? (I'm not quite sure what "in row" meant here)

Reply:
Yes, I mean draw a horizontal line here, the old version here is direction == 0 or direction == 1 here, but i feel ambiguity here. Then i create the Axis class to do this.forget to rewrite the comment here. your comment applied

Reply:
Thanks for the explanation! Agreed that writing `Axis.horizontal` is much clearer than just `1`